### PR TITLE
Release 3.2.3

### PR DESCRIPTION
**Pull request by Goose-Of-War**
Git Hook Fix

### CODE REVIEW

**Comment by Goose-Of-War** [comment]
Finally, git-hook stuff is done with (hopefully) and some misc things were also added

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The website for the Manga & Anime Society Kharagpur
 
 The owner of this repository is @PartMan7, who also runs the server. He's also running the server cost (at the time of writing he's in his final year, but it'll carry on even after graduation).
 
-All non-trivial changes are done through PULL REQUESTS ONLY. The WebDev Team Head is responsible for testing and merging all PRs. Feel free to pester them to look at the changes you've prepared.
+All non-trivial changes are done through PULL REQUESTS ONLY. The WebDev Team Head (currently @Goose-Of-War) is responsible for testing and merging all PRs. Feel free to pester them to look at the changes you've prepared.
```
Comment (nishkalprakash) [inline README.md:8]: Congratulations,
party webd team head!

Comment (nishkalprakash) [reply README.md:8]: @ItsAnkan @SachdevJai @soumil

Comment (Goose-Of-War) [reply README.md:8]: nou

Comment (ItsAnkan) [reply README.md:8]: Yes

Comment (Goose-Of-War) [reply README.md:8]: > @ItsAnkan @SachdevJai @soumil
> 

You owe us a party -_-

File: README.md [hunk 2]
```diff
@@ -125,6 +125,7 @@ Take a look at existing articles for the various classes and where they're used.
 - <a href="https://github.com/MokshithPV" target="_blank">Venkatsai Mokshith</a> (Contributor)<br />
 - <a href="https://github.com/SachdevJai" target="_blank">Jai Sachdev</a> (Contributor)<br />
 - <a href="https://github.com/ItsAnkan" target="_blank">Ankan Saha</a> (Contributor)<br />
+- <a href="https://github.com/Symbiot01" target="_blank">Sahil Patel</a> (Contributor)<br />
```
Comment (nishkalprakash) [inline README.md:128]: didn't soumil also contribute?

**Review by nishkalprakash — APPROVED** [review]
Perfect!

you may add soumil or you can proceed.

nothing else from my end.

File: README.md (hunk 2 above)
Comment (Goose-Of-War) [reply README.md:128]: Not yet

**Pull request merged**